src/Directory.Build.targets

```diff
@@ -7,6 +7,18 @@
     <ErrorReport Condition="'$(ErrorReport)' == 'prompt'" />
     <WarningsAsErrors Condition="'$(WarningsAsErrors)' == 'NU1605'" />
   </PropertyGroup>
+  
+  <!-- Adds Nullable annotation attributes to netstandard <= 2.0 builds -->
+  <Choose>
+    <When Condition="'$(Nullable)' != '' and ($(TargetFramework.StartsWith('netstandard1')) or '$(TargetFramework)' == 'netstandard2.0' or $(TargetFramework.StartsWith('netcoreapp2')) or '$(TargetsNetFx)' == 'true')">
```

Comment:
What are the possible values of Nullable?

Reply:
`enable`, `restore`, `disable`... I would use `'$(Nullable)' == 'enable'` as the condition instead.

Reply:
The change here is intentional. Some projects depend on the nullability attributes but are not yet annotated. I've explicitly set Nullable to `annotations` for those projects so that the logic here is triggered. c.f. [this commit](https://github.com/dotnet/corefx/pull/42391/commits/75b76b54199909131edc36d0f628c28252584870) for more context.

Reply:
I see can we instead be explicit and check for those two values? What if nullable is set to disable and we keep including this source file into its compile items?

Reply:
That's a good point, but do we realistically expect `disable` to appear in corefx given that it's turned off by default?

Reply:
Note that we'd need to support `warnings` too, so the truly correct approach would require checking that `Nullable` is defined and also not equal to `disable`.

Reply:
That sounds like the right thing to do.

Reply:
Agreed. I don’t expect a project to have disable, but we need to account for it, we can’t guarantee there will be no projects like that.

Reply:
I'm not sure. Explicitly setting a `Nullable` property probably means that the project uses nullability one way or another. It is very likely that the project explicitly disables it but the dependency on attributes is still there (e.g. because it is enabled by pragmas in individual source files).

Reply:
Resolved as per offline discussion as we expect the nullable disable scenario to not be present in corefx 🤞